features/order_items.feature

```diff
@@ -21,3 +31,17 @@ Feature: The order service back-end
         And I set the "Quantity" to "3"
         And I press the "Create Item" button
         Then I should see the message "Success"
+
+    Scenario: List Items
+        When I visit the "Home Page"
+        And I press the "Clear" button
+        And I press the "Search" button
+        Then I should see the message "Success"
+        When I press the "Clear Item" button
```

Comment:
after clear btn, also have to check whether all the fields are empty.

Reply:
OK